src/main/java/binnie/extrabees/config/EBConfigGUIFactory.java

```diff
@@ -0,0 +1,13 @@
+package binnie.extrabees.config;
+
+import net.minecraft.client.gui.GuiScreen;
+
+import com.gtnewhorizon.gtnhlib.config.SimpleGuiFactory;
+
+public class EBConfigGUIFactory implements SimpleGuiFactory {
```

Comment:
Are these factories (two of them) doing anything special? They are no longer needed for gtnhlib configs if the standard gui works for what you want